src/realm/group_shared.cpp

```diff
@@ -1326,10 +1326,14 @@ bool SharedGroup::compact()
             bool disable_sync = get_disable_sync_to_disk();
             if (!disable_sync)
                 file.sync(); // Throws
-            }
```

Comment:
@finnschiermer that's why I said I'd like the extra braces in that Table method the other day :D